README.md

```diff
@@ -1,2 +1,65 @@
-# module-ballerina-regex
-Ballerina Regex Module
+Ballerina Regex Library
+==============================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-regex/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-regex/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-regex.svg)](https://github.com/ballerina-platform/module-ballerina-regex/commits/main)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/regex.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fregex)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+This Regex library is one of the standard libraries of <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+
+For a quick sample on demonstrating the usage see [Ballerina By Example](https://ballerina.io/swan-lake/learn/by-example/).
```

Comment:
Shall we remove this. since we don't have a BBE to point. will add this when we add a BBE
```suggestion
```

Reply:
Shall we have a description of this module. Explaining what we support in this module